packages/node_modules/@webex/webex-core/src/interceptors/default-options.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {Interceptor} from '@webex/http-core';
+
+
+/**
+ * Allows the user of the SDK to set default options that apply every http request made by the SDK
+ * For example a default timeout could be set for all requests like this :
+ *
+ *   webex = WebexSdk.init({
+ *     credentials: {
+ *       supertoken: superToken
+ *     },
+ *     config: {
+ *       credentials: {
+ *         client_id,
+ *         client_secret
+ *       },
+ *       defaultRequestOptions: {
```

Comment:
I like how this can be implemented at the instance level. Might be able to shorten this to just `requestOptions`. Need additional input from the team.

Reply:
`defaultRequestOptions` works for me and is descriptive enough to not be confusing